Add October 2025 newsletter (#2450)
writethedocs/www: docs/blog/newsletter-october-2025.rst

:og:image: https://www.writethedocs.org/_static/logo-opengraph.png

.. post:: October  06, 2025
  :tags: newsletter

########################################
Write the Docs Newsletter – October 2025
########################################

Hello, documentarians! I hope you are all staying grounded amid a world that seems to be changing fast. Docs still matter and so do you.

In conference news, Write the Docs Berlin is coming up soon, on October 27-28. If you don't have your in-person ticket yet, `get one soon </conf/berlin/2025/tickets/>`__ before they sell out. Or if you can't be there in person, get a virtual ticket for live access to the talks and interaction with other attendees. 

> If you're already looking forward to next year, Write the Docs Portland 2026 is happening May 3–5. See `more info </conf/portland/2026/news/welcome/>`__ and start making plans.

Author: ```suggestion
Looking ahead to next year, Write the Docs Portland 2026 is happening May 3–5. See `more info </conf/portland/2026/news/welcome/>`__ and start making plans.
```

Reviewer: Thanks!

> If you're already looking forward to next year, Write the Docs Portland 2026 has its dates set for May 3–5. See `more info </conf/portland/2026/news/welcome/>`__ and start making plans.

Reviewer: Wanna reword the passive here?

Author: ```suggestion
If you're already looking forward to next year, Write the Docs Portland 2026 is happening May 3–5. See `more info </conf/portland/2026/news/welcome/>`__ and start making plans.
```

Reviewer: Oh, and now I noticed one of my pet peeves as well (aka, ignore or not as you will).

WTD Portland is happening whether you're already looking forward to it or not :-D :-D

Reviewer: But the change looks good

Author: A very fair point.

In other news, the `2025 Salary Survey <https://salary-survey.writethedocs.org/>`__ is now open for submissions. We encourage contributions from everyone who has worked in a documentation-related role for at least part of the past year -- full-time, part-time, employee, contractor, or any combination thereof. Returning contributors will be pleased to hear that it's now shorter! 

> This month's newsletter has articles on whether AI mean people don't need to know the product, how to encourage others at your company to search before asking, how to use AI to prove your value, and whether to publish docs late or push to delay releases. Enjoy!

Reviewer 2: ```suggestion
This month's newsletter has articles on whether AI means people don't need to know the product, how to encourage others at your company to search before asking, how to use AI to prove your value, and whether to publish docs late or push to delay releases. Enjoy!
```

-------------------------------------
AI and the need for product knowledge
-------------------------------------

As generative AI and AI agents change how users find help, documentarians might be wondering whether users still need traditional product knowledge from documentation to use software effectively.

> AI agents can reduce the need for product knowledge but do not eliminate it. Users tend to see how far they can get before they get stuck, but they typically do eventually need the docs. Users also still need a baseline understanding of the product and its capabilities. LLMs themselves rely on documentation to provide accurate information. Documentation therefore remains the trusted source of truth for users as well as AI assistants that are trained on product content.

Reviewer: The last sentence feels like a very broad statement to make.

> Documentation therefore remains the trusted source of truth for users as well as AI assistants that are trained on product content.

Author: ```suggestion
AI agents can reduce the need for product knowledge but do not eliminate it. Users tend to see how far they can get before they get stuck, but they typically do eventually need the docs. Users also still need a baseline understanding of the product and its capabilities. LLMs themselves rely on documentation to provide accurate information. Documentation therefore remains a reliable source of info for users and AI assistants trained on product content.
```

Development moves quickly. LLMs that can interpret codebases and summarize product capabilities and new features may become available. However, accuracy is likely to remain a concern and such LLMs would still need to be primed with good documentation.

See more Write the Docs resources about `AI and LLMs </topics/#ai-and-llms>`__.

--------------------------------------------
Encouraging colleagues to search for content
--------------------------------------------

If you (or your team) are responsible for creating content for internal audiences, how do you get fellow employees to read the documentation? A recent conversation brought up frustrations about this problem. First, consider asking whether employees have problems finding the content they need. This could be valuable feedback about the existing documentation – for example, ease of use or missing content.

> In general, it’s unsatisfactory to always answer questions when documentation is available. Others may prefer direct answers to their questions, but it reinforces bad habits and can overwhelm your team and impede your team’s work.

Reviewer: `s/In general//` ?

Author: ```suggestion
It’s unsatisfactory to *always* answer questions when documentation is available. Others may prefer direct answers to their questions, but it reinforces bad habits and can overwhelm your team and impede your team’s work.
```

See more Write the Docs resources about `working with other roles </topics/#working-with-other-roles>`__.

------------------------------------------
Using AI to prove AI shouldn’t replace you
------------------------------------------

Many organizations are adopting AI tools to help users find information faster. As these tools become more common in documentation workflows, some leadership teams are beginning to question whether human writers are still needed. In some cases, they have even explored using AI to automate routine updates or replace documentarians altogether.

Ironically, AI can help prove why documentation teams are essential. AI-powered search tools, such as Kapa, can estimate the return on investment (ROI) of documentation through metrics such as support hours saved and customer frustration avoided. These numbers turn the usually invisible impact of documentation into something concrete that leadership can understand. When tracked and shared regularly, they can influence critical business decisions – such as which roles to keep during layoffs or where to invest resources.

> It’s also important to remember that AI depends on high-quality, current content. If documentation is outdated or incomplete, AI will surface wrong answers, frustrate users, and damage trust. AI can scale access to good content, but it cannot maintain or verify that content on its own.

Author: ```suggestion
It’s also important to remember that AI depends on high-quality, current content. If documentation is outdated or incomplete, AI will provide wrong answers, frustrate users, and damage trust. AI can make it easier for more people to access to good content, but it cannot maintain or verify that content on its own.
```
